<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>-10</y>
      <width>251</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>16</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>321</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3" colspan="3">
      <widget class="QPushButton" name="pushButtonDisconnect">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="6">
      <widget class="QLineEdit" name="lineEditIP"/>
     </item>
     <item row="3" column="4" colspan="2">
      <widget class="QLCDNumber" name="lcdNumberMax"/>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QSlider" name="horizontalSliderMin">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Timing(s)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4" colspan="2">
      <widget class="QLCDNumber" name="lcdNumberTime"/>
     </item>
     <item row="5" column="3" colspan="3">
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QSlider" name="horizontalSliderMax">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4" colspan="2">
      <widget class="QLCDNumber" name="lcdNumberMin"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QPushButton" name="pushButtonConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QSlider" name="horizontalSliderTime">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTime</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberTime</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
